src/stylus/fablevision/trial-count.styl

```diff
@@ -0,0 +1,25 @@
+.geniblocks.trial-count
+  width: 158px
+  height: 107px
+  background-image: url('../resources/fablevision/huds/Trial_screen.png')
+  background-size: 100% 100%
```

Comment:
Interesting, I've been using contain...either solution requires us looking up the size of the image, which doesn't feel great. Is there a better way to do this?